src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLikes }) => {
+  const handleLike = () => {
+    updateLikes(id);
+  };
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className="chat-entry">
```

Comment:
👀 Note that without `local` as part of the style, the chat bubbles lose the little pointy bit to indicate the source of the speech.